### ISSUE TITLE
Static routes in fabio.properties

### ISSUE THREAD

**Issue by nuriel77** [fabiolb/fabio#498]
I am trying to add static routes in the fabio.properties configuration file.

I tried:
```
registry.static.routes = \
    route add svc 10.70.0.23/ http://google.com:80/
```
Or
```
registry.static.routes = \
    route add svc my-test.example.com/ http://google.com:80/
```
Calling via:
```
curl -H "Host: my-test.example.com" http://10.70.0.23/
```
But I get in the logs:
```
[WARN] No route for 10.70.0.23/
```
or:
```
[WARN] No route for my-test.example.com/
```
When using the hostname.

After restarting fabio I check the logs, and I can see:
```
May 17 17:28:33 lbtest-1.localdomain fabio[1422]: "Static": {
May 17 17:28:33 lbtest-1.localdomain fabio[1422]: "Routes": "route add svc 10.70.0.23/ http://google.com:80/"
May 17 17:28:33 lbtest-1.localdomain fabio[1422]: },
```
In any case I cannot see the route in output of:
```
curl  http://10.70.0.23:9998/api/routes
```

I wonder if I am doing something wrong?

**Comment by nuriel77** [comment]
Ok, I see that it is not possible to combine both consul and static routes, as this is what I initially tried to do.

**Comment by vjeantet** [comment]
Hello, how did you manage to combine routes coming from consul, with additional static routes ?

I did not find any solutions on issues, docs and google, thanks a lot !

**Comment by nuriel77** [comment]
@vjeantet as noted in my last comment: I did **not** find a way to do that. If I understand correctly, this is not possible.

**Comment by tristanmorgan** [comment]
I wonder if Consul-ESM could help with adding the "services" with the right tags?...

**Comment by leprechau** [comment]
@vjeantet @nuriel77 You can add static routes within consul itself.  When you are using the `consul` backend you can add route information via the KV store specified with the `registry.consul.kvpath` setting in the properties file.

Example KV entry to redirect all HTTP traffic to HTTPS on a particular domain:
```
route add http-redirect fbdev.ena.net:80 https://fbdev.ena.net$path opts "redirect=301"
```

**Comment by nuriel77** [comment]
@leprechau @vjeantet Thanks. I completely forgot this is how I solved it (solution is already in production for a while now).
The reason I initially said I thought it was not possible was because I tried to set static route in the fabio.properties file.

I can confirm that I am setting static routes via consul KV along side having fabio read routes from Consul services.

What I have done was edit the systemd control file (/etc/systemd/system/fabio.service) to run a ExecStartPost command to set a static route I had to add:

```
ExecStartPost=/usr/bin/curl -X PUT -H "Content-Type: application/json" -k http://consul:8500/v1/kv/fabio/config -d "route add svc health.app.some.domain/ https://app.other.domain/"
ExecStart=/opt/fabio/bin/fabio $OPTIONS
```
